Use the sql generic 'Limit Offset' syntax, not MySQL only one

John71-Off opened the pull request:
Using `LIMIT a OFFSET b` on `\FaaPz\PDO\Clause\Limit#__toString` that is the generic Limit syntax instead of `LIMIT a, b` which is only for MySQL

(You can take for reference the [sql.sh](https://sql.sh/cours/limit) website or this other one: [https://www.sqltutorial.org/sql-limit/](https://www.sqltutorial.org/sql-limit/))

_Thank you !_

John71-Off committed:
Use the sql generic 'Limit Offset' syntax, not MySQL only one

Using `LIMIT a OFFSET b` on `#__toString` that is the generic Limit syntax instead of `LIMIT a, b` which is only for MySQL

kwhat commented:
Hi @JohnProgrammer71, 

The change should be ok, can you fix the failing tests?

Thanks.

kwhat commented:
https://github.com/FaaPz/PDO/blob/2.x/tests/Statement/SelectTest.php#L161

John71-Off committed:
Update SelectTest.php

Update #testToStringWithLimit to apply change made on FaaPz\PDO\Clause\Limit#__toString

kwhat commented:
I am pulling this in with PR #153 due to the tests still failing on this PR.
